recipes/ca-certificates/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Create the directory to hold the certificates.
+mkdir -p "${PREFIX}/ssl"
+
+# Copy the certificates from certifi.
+# It is ok to hard-code Python here as we fixed the version before.
+cp -f "${PREFIX}/lib/python3.5/site-packages/certifi/cacert.pem" "${PREFIX}/ssl"
```

Comment:
We can just use ```SP_DIR``` (see http://conda.pydata.org/docs/building/environment-vars.html?highlight=environment%20variables) - we don't even need the comment at that point.